benchmarks/user/benchmark.py

```diff
@@ -409,12 +408,9 @@ def param_string(self, params):
             else:
                 devito_params['nt'] = 1
 
-            if configuration['mpi']:
-                devito_params['np'] = MPI.COMM_WORLD.size
-                devito_params['rank'] = MPI.COMM_WORLD.rank
-            else:
-                devito_params['np'] = 1
-                devito_params['rank'] = 0
+            devito_params['np'] = MPI.COMM_WORLD.size
```

Comment:
this fails if one doesn't have mpi4py installed
https://github.com/opesci/devito/blob/master/devito/mpi/distributed.py#L39

Reply:
Sure, thanks for testing.